plugins/cost-insights/README.md

```diff
@@ -178,6 +178,21 @@ costInsights:
       name: Metric C
 ```
 
+### Base Currency (Optional)
+
+In the case you would like to show your baseline costs on the graph on other currency than US dollars.
+
+```yaml
+## ./app-config.yaml
+costInsights:
+  engineerCost: 200000
+  baseCurrency:
+    locale: en-US
+    options:
+      currency: EUR
```

Comment:
locale & currency should align in example as well :)

Reply:
still the case - unresolved conversation

Reply:
🙈